packages/cadl-ranch-specs/http/azure/example/basic/client.tsp

```diff
@@ -0,0 +1,50 @@
+import "@azure-tools/typespec-client-generator-core";
+
+import "./main.tsp";
+
+using Azure.ClientGenerator.Core;
+
+@TypeSpec.Versioning.useDependency(_Specs_.Azure.Example.Basic.Versions.v2022_12_01_preview)
+namespace Client;
+
+@client({
```

Comment:
a little bit confused why we use client.tsp to add example?

Reply:
I'd like the case verify the correct match of ServiceOperation and JSON example via operationId = `ServiceOperationGroup_Basic`

Just a test to avoid picking up the wrong operationId like `AzureExampleClient_basicAction` from the explicit client.

Reply:
oh, i see.